### PR TITLE
[13.0][fix] base_tier_validation

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Forward port of https://github.com/OCA/server-ux/pull/112.

cc @ps-tubtim 

I also use this PR to update copyright on the module after rebranding of Eficent to ForgeFlow.

### CODE REVIEW

File: base_tier_validation/tests/test_tier_validation.py [hunk 1]
```diff
@@ -227,6 +227,11 @@ def test_11_add_comment(self):
         wiz = wizard.save()
         wiz.add_comment()
         self.assertTrue(test_record.review_ids.mapped("comment"))
+        # Check notify
+        comment = test_record.sudo(self.test_user_1.id)._notify_accepted_reviews_body()
+        self.assertEqual(comment, "A review was accepted. (Test Comment)")
+        comment = test_record.sudo(self.test_user_1.id)._notify_rejected_review_body()
+        self.assertEqual(comment, "A review was rejected by John. (Test Comment)")
```
Comment (HviorForgeFlow) [inline base_tier_validation/tests/test_tier_validation.py]: r/sudo/with_user

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: base_tier_validation/tests/test_tier_validation.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply base_tier_validation/tests/test_tier_validation.py]: done!

**Review by HviorForgeFlow — APPROVED** [review]
:+1:

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by ps-tubtim — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Pull request merged**